server/shared/database/migrations/20190606145833-add-revision-enum-value.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const replaceEnum = require('sequelize-replace-enum-postgres').default;
```

Comment:
I knew I had it somewhere:
https://github.com/ExtensionEngine/boutique/blob/9ae920/server/common/database/helpers.js#L12-L19